core/backend/src/TextAnnotationLayout.ts

```diff
@@ -79,9 +83,20 @@ export function layoutTextBlock(args: LayoutTextBlockArgs): TextBlockLayout {
   return new TextBlockLayout(args.textBlock, new LayoutContext(args.textBlock, computeTextRange, findTextStyle, findFontId));
 }
 
+/**
+ * Gets the result of laying out the the contents of a TextBlock into a series of lines containing runs.
+ * The visual layout accounts for the [TextStyle]($common)s, fonts, and [TextBlock.width]($common). It applies word-wrapping if needed.
+ * The layout returned matches the visual layout of the geometry produced by [[produceTextAnnotationGeometry]].
+ * @beta
+ */
+export function getLayoutTextBlockResult(args: LayoutTextBlockArgs): TextBlockLayoutResult {
```

Comment:
Nit, but "get" implies a trivial operation, "compute" would be more appropriate for a function that does a whole bunch of computations.